docs/wiki/_DELETED/Install-Script-for-16.X.md

```diff
@@ -27,7 +27,7 @@ For CentOS 6 systems a separate [[CentOS Readme|https://github.com/aces/Loris/bl
 ### Mac OSX
```

Comment:
modifying files from `_DELETED` is not very useful, these files will be compt=letely deleted once the full wiki is released